conf/README.md

```diff
@@ -0,0 +1,123 @@
+# Config directory
+
+In this directory we store all the configuration for clusters and OCSCI config
+files.
+
+During the execution, via
+[ocsci pytest plugin](../pytest_customization/ocscilib.py), we are loading all
+the configs described below, combining them togehter and using
+[jinja2](http://jinja.pocoo.org/docs/2.10/templates/#variables)
+templates rendering variables which you can reuse among different configs from
+DEFAULT and different sections.
+
+This gives us the ability to easily reuse variables from context of the
+yaml file itself.
+
+Cause you can overwrite data just in specific section like for ENV_DATA
+(see below) and if you don't overwrite data in DEFAULT section, someone can
+still access old default value from DEFAULT section directly. See example of
+[this cluster config file](./examples/ocs_basic_install.yml) for more details.
+
+## OCS CI Config
+
+We moved all the OCSCI framework related config under
+[ocsci folder](./ocsci/). You can see we have defined here:
+
+* [default_config.yaml](ocsci/default_config.yaml) - this is the default
+    config file for OCS CI which is loaded automatically (don't pass it to
+    --ocsci-conf). It contains multiple sections which you can overwrite in
+    some custom config file you create or in feature we will have more config
+    files defined under `ocsci` folder as well which you can reuse.
+
+## Cluster config
+
+If you would like to overwrite cluster default data you can create file
+similar to [this example](./examples/ocs_basic_install.yml). You can see we
+can overwrite default data like:
+
+* `platform` - currently we support only AWS, but in future we will cover
+    more.
+* `worker_replicas` - number of replicas of worker.
+* `rook_image` - which rook image can be used.
+* and many more - look at the deployment templates under
+    [../templates/ocs-deployment/](../templates/ocs-deployment/) and you can
+    see the variables in templates. In the deploment we are using ENV_DATA
+    section as data for rendering the templates whic allow us to add new
+    values just in config and we don't have to also update the code on all
+    places to take specific variable.
+
+If you define cluster config you have to put all the data under `ENV_DATA`
+section like in the mentioned example. Actually you can add another section as
+well but in cluster config file we recommend to define just `ENV_DATA` section
+or one more additional section which is `DEFAULTS` for propagation those
+defaults among all configs as well.
+
+### Sections in our configs
+
+All of the sections below, except for `DEFAULTS` one, will be available from
+[ocsci/config.py](../ocsci/config.py) module. `DEFAULTS` section is exposed to
+[ocs/defaults.py](../ocs/defaults.py) module as described below.
+
+#### DEFAULTS
+
+Default values which are used among different configs and sections.
+
+All **capital letters** defined variables are exposed to
```

Comment:
why just CAPS, why not all? you probably want to simple the next 7 lines and just say "Defaults will be exported via defaults.py"

Reply:
In the defaults we would like to have mainly constants. So if we follow pep8 convention constants are with CAPS. I can put there everything, cause I solved the issue with loading  this config which I will send in new commit.